dmipy/utils/build_sphere.py

```diff
@@ -0,0 +1,37 @@
+import numpy as np
+from dipy.core.sphere import disperse_charges, HemiSphere
+
+
+def get_hemisphere(directions=1000):
```

Comment:
Did you write this function from scratch? It looks quite nice, but how does it compare with just loading a default tesselation?

Reply:
blabla

Reply:
I removed it. We can do the same thing the csd optimizer does